services/learn-card-network/brain-service/src/helpers/notifications.helpers.ts

```diff
@@ -0,0 +1,97 @@
+interface IPushNotificationRequest {
+    to: string;
+    title: string;
+    message?: string;
+    url?: string;
+    actionType: string;
+}
+
+interface IPushRegistrationRequest {
+    profileId: string;
+    deviceToken: string;
+}
+
+export async function SendPushNotification(args: IPushNotificationRequest) {
+    try {
+        if (process.env.WEBHOOK_URL) {
+            const response = await fetch(process.env.WEBHOOK_URL, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'apikey': `${process.env.APIKEY}`,
```

Comment:
Same here, and should we rename to NOTIFICATIONS_SERVICE_API_KEY to disambiguate?

Reply:
changed